language/stdlib/modules/validator_config_2.move

```diff
@@ -95,8 +96,6 @@ module ValidatorConfig2 {
         // TODO(valerini): verify the proof of posession of new_consensus_secretkey
 
         let t_ref = borrow_global_mut<T>(validator_account);
-        // Assert that the new key is different from the old key
```

Comment:
why not? is there value in rotating in the same key?

Reply:
Just that some tests written prior to this change expect this function to succeed, not abort in case the new key is the same. Do you think, we should abort here? The effect of the function is the same, just the end-status of the function will be different...